src/test/java/com/google/api/codegen/testdata/python_doc_types_library.baseline

```diff
@@ -0,0 +1,65 @@
+============== file: google/cloud/proto/example/library_v1/types.py ==============
```

Comment:
To be clear, the back-incompat change is fine. I am willing to go update the world to get this change. :-)